packages/caraml-auth-google/README.md

```diff
@@ -0,0 +1,4 @@
+# CaraML Google Authentication
+
+Utility package containing functions to authenticate users of the Merlin and Turing SDKs of CaraML.
```

Comment:
Nit: Might be better to say "authenticate users of the Python SDKs of CaraML." than explicitly say Turing and Merlin. There is a chance, at some point, we could integrate other components (like DAP?) and right now, from this repo's POV, there is no explicit connection to Turing/Merlin with this repo.

Reply:
Also, some simple dev notes on installation (as of now, using the git repo ref), tests, etc. will be useful.

Reply:
Ok I've included additional dev docs on the usage and testing!